src/elements/content-sidebar/activity-feed/comment/CreateReply.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import * as React from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import type { IntlShape } from 'react-intl';
+import noop from 'lodash/noop';
+
+import PlainButton from '../../../../components/plain-button';
+import ArrowArcRight from '../../../../icon/fill/ArrowArcRight';
+
+import type { SelectorItems } from '../../../../common/types/core';
+
+import CommentForm from '../comment-form';
+
+import messages from './messages';
+import './CreateReply.scss';
+
+type Props = {
+    getMentionWithQuery?: (searchStr: string) => void,
+    intl: IntlShape,
```

Comment:
Do we have to include this in the props? Could we just import `intl` to use below?